google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/prepare/constants.py

```diff
@@ -16,9 +16,12 @@
 
 ENTRY_ID_LENGTH = 64
 ENTRY_ID_PREFIX = 'qlik_'
-ENTRY_ID_STREAM = 'st_'
+ENTRY_ID_APP = 'app_'
+ENTRY_ID_STREAM = 'str_'
```

Comment:
Shouldn't the constants be named  `ENTRY_ID_APP_PREFIX` and `ENTRY_ID_STREAM`?

Reply:
They are not the prefix themselves, but part of the generated IDs. I've renamed some constants and added comments to clarify the intended usage. Please let me know if it's clearer now.

Reply:
I think this is confusing, the constant name gives the impression that they are the ID, but actually they are just part of the ID.

Adding the prefix makes it easier to understand the code.

Looking at the code:
```
generated_id = self.__format_id(constants.ENTRY_ID_STREAM,
                                        app_metadata.get('id'))
```

```
def __format_id(cls, source_type_prefix, source_id):
...
```
So the parameter `source_type_prefix` received `constants.ENTRY_ID_STREAM`. This is being used as a prefix.